suites/pacific/cephfs/tier-4_fs.yaml

```diff
@@ -0,0 +1,149 @@
+---
+#=======================================================================================================================
+# Tier-level: 2
+# Test-Suite: tier-4_fs
+# Conf file : conf/pacific/cephfs/tier_0_fs.yaml
+# Test-Case Covered:
+# CEPH-83574833 - Performace test for file write_sync() on cephfs kernel mount
+#=======================================================================================================================
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: cluster deployment
+      desc: Deploy cluster with enforcing mode(default mode).
+      module: test_cephadm.py
+      polarion-id: CEPH-83573740
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                registry-url: registry.redhat.io
```

Comment:
Could we remove this `key/value` as we use a different registry when executing in IBM environment. It's recommended to have environment agnostic suites unless this needs to be executed in PSI environment only.

Reply:
Okay